7_deploy_osh/roles/suse-osh-setup/defaults/main.yml

```diff
@@ -4,4 +4,15 @@
 # Location of the kubeconfig file, fetched from velum UI.
 kubeconfig_file_path: "~/suse-osh-deploy/kubeconfig"
 
-suse_cp_node_create_subvolume: false
\ No newline at end of file
+suse_cp_node_create_subvolume: false
```

Comment:
This can be ignored, as the steps here are taken in https://github.com/SUSE-Cloud/socok8s/pull/93 .